### PR TITLE
Additional Form Group Attributes in select example

### PR DESCRIPTION
**Pull request by r-sierra**
Show how to set additional Form Group Attributes like wrapper class

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ You still can use `wrapper_class` option to set only a css class. This is just a
 Our select helper accepts the same arguments as the [default Rails helper](http://api.rubyonrails.org/classes/ActionView/Helpers/FormOptionsHelper.html#method-i-select). Here's an example of how you pass both options and html_options hashes:
 
 ```erb
-<%= f.select :product, [[1, "Apple"], [2, "Grape"]], { label: "Choose your favorite fruit:" }, { class: "selectpicker" } %>
+<%= f.select :product, [[1, "Apple"], [2, "Grape"]], { label: "Choose your favorite fruit:" }, { class: "selectpicker",  wrapper: { class: 'has-warning', data: { foo: 'bar' } } %>
```
Comment (lcreid) [inline README.md]: Note that there was a typo in the original README.md. PR #355 will fix the typo, at which time this line will probably have a merge conflict. Please wait for the other PR to be merged, fix the conflict if any, and then we can merge this PR.

**Comment by mattbrictson** [comment]
@r-sierra would you mind rebasing against the latest `master` and resolving the conflict? Thanks!

**Comment by r-sierra** [comment]
@mattbrictson conflict solved!

**Comment by mattbrictson** [comment]
Much appreciated! 🙇

**Pull request merged**